### ISSUE TITLE
ssh client in terminal

### ISSUE THREAD

**Issue by lucj** [HeyPuter/puter#72]
I'm discovering Puter, this is a great project.
Navigating in the destkop, I run the terminal and tried to ssh to a remote host but got an error as there is no ssh client available.
Is this something planned ?

**Comment by jelveh** [comment]
Yes. It's probably going to be at least a few weeks from now, but yes. I'm moving this to the Phoenix Shell project.
